src/components/schemas/reimbursement.yaml

```diff
@@ -105,4 +105,29 @@ reimbursement_in_minimal:
         This id is always prefixed with `reim`.
       example: reimntGt9ju6qP
     code:
-      $ref: './fields.yaml#/code'
\ No newline at end of file
+      $ref: './fields.yaml#/code'
+
+reimbursement_out_minimal:
+  type: object
+  properties:
+    id:
+      type: string
+      maxLength: 255
+      description: |
+        This id provided by Fyle to uniquely identify a reimbursement.
+        This id is always prefixed with `reim`.
+      example: reimntGt9ju6qP
+    currency:
+      $ref: './fields.yaml#/currency'
+    amount:
+      $ref: './fields.yaml#/amount'
+    mode:
+      type: string
+      enum:
+        - ACH
+        - OFFLINE
+      description: |
+        Mode of creation of this reimbursement.
+      example: ACH
+    ach_transfer:
```

Comment:
this is a new field right ? should be added to main reimbursements.yaml if correct